triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/TriggerRunnerImpl.scala

```diff
@@ -83,9 +132,14 @@ object TriggerRunnerImpl {
               // is sent to a now terminated actor. We should fix this
               // somehow™.
               ctx.pipeToSelf(trigger) {
-                case Success(_) => Failed(new RuntimeException("Trigger exited unexpectedly"))
-                case Failure(cause) => Failed(cause)
+                case Success(_) =>
+                  Failed(new RuntimeException("Trigger exited unexpectedly"))
+                case Failure(cause) =>
+                  Failed(cause)
```

Comment:
These two `Failed` cases are where the parent will use the retry strategy (without being stopped by the Server). Is that right?

Reply:
yes